CONTRIBUTING.md

```diff
@@ -1,193 +1,49 @@
+Contributions to ProbNum are very welcome! Before getting started make sure to read the following guidelines.
 
-All contributions to ProbNum should be made via pull requests (PR) to the
-[master branch](https://github.com/probabilistic-numerics/probnum/tree/master) on GitHub. Some suggestions for
-a good PR are:
+Some suggestions for contributions:
 
-- implements or fixes one functionality;
-- includes tests and appropriate documentation; and
-- makes minimal changes to the interface and core codebase.
+- adding documentation
+- adding tutorials
+- requesting a feature
+- implementing a new feature
+- reporting a bug
+- fixing a bug
+- adding missing tests
 
-If you would like to contribute but are unsure how, then writing examples, documentation or working on
-[open issues](https://github.com/probabilistic-numerics/probnum/issues) are a good way to start. See the
-[developer guides](https://probnum.readthedocs.io/en/latest/development/developer_guides.html)
-for detailed instructions.
+If you would like to contribute but are unsure how, then writing turorials, documentation or working on
+[open issues](https://github.com/probabilistic-numerics/probnum/issues) are a good way to start. 
 
-## Getting Started
+## Reporting a bug
+If you find a bug, please report it by opening an issue on the GitHub page, and tag it as a bug.
+Reproducibility is key, so if possible provide a working code snipped that produces the bug, and tell us the python 
+version and ProbNum version you used.
 
-Begin by forking the repository on GitHub and cloning your fork to a local machine.
+Please also detail how you found the bug, what you expected to happen, and what happened instead. Have you made any 
+code changes after which the bug occurred?
 
-```shell
-git clone git@github.com:MyGithubAccount/probnum.git
-```
+Of course, bug-fixes via pull request are very welcome as well.
 
-Next, create a new branch in your
-forked repository describing the feature you would like to implement.
-```shell
-git checkout -b my-new-feature
-```
-You can now get started writing code in your new
-branch. Make sure to keep the following best practices regarding code quality in mind.
+## Requesting a feature
+If you are working with ProbNum, and you are missing a feature, please open an issue and detail what feature you 
+would like to see. Please also check existing issues in case someone else requested the same or a similar feature. 
+Detail the indented functionality and use case of the feature. 
+You can also lay out how you would implement the feature. 
+If you have a working version, please consider doing a pull request!
 
-### Code Quality
+## Contributing via pull request
 
-Code quality is an essential component in a collaborative open-source project.
+You implemented an additional feature, a tutorial, tests, or some other enhancement to ProbNum? That's great! 
+Please consider doing a pull request (PR) of the changes. 
```

Comment:
opening a pull request (PR) with the changes

Reply:
done